Dockerfile.fedora-25-master-nightly

```diff
@@ -53,6 +53,7 @@ RUN systemctl enable ipa-server-configure-first.service
 COPY exit-via-chroot.conf /usr/lib/systemd/system/systemd-poweroff.service.d/
 
 RUN groupadd -g 288 kdcproxy ; useradd -u 288 -g 288 -c 'IPA KDC Proxy User' -d '/var/lib/kdcproxy' -s '/sbin/nologin' kdcproxy
+RUN groupadd -r ipaapi ; useradd -g ipaapi -c 'IPA Management Framework User' -d '/var/lib' -s '/sbin/nologin' ipaapi
```

Comment:
Given that FPC was not keen on allocating uids for container use and they are also opposed to allocating them in general, per recommendation in https://pagure.io/packaging-committee/issue/474 we might need to either find someone to manage the list for us, or self-proclaim us to be the list maintainers.